_lessons/maui-and-mahuika/01-introduction.md

```diff
@@ -13,4 +13,98 @@ You will learn:
+
+Follow the blue network connections for user-accessible nodes. All Mahuika CS400 nodes and Māui CS500 nodes can access Spectrum Scale storage natively via InfiniBand for fast IO (red lines). Cray Aries interconnect (green lines) is used for low-latency communication between XC50 compute nodes on Māui, and for Spectrum Scale access via DVS (Data Virtualization Service) nodes.
+
+#### Mahuika Compute Hardware:
+##### 234 CS400 compute nodes
+* 2x Intel Xeon E5-2600 v4 Broadwell 2.1 GHz processors with 18 physical cores each (36 physical cores per node)
+* 128 GB of RAM per node
+
+##### 16 CS400 large memory nodes
+* Same Intel Xeon E5-2600 v4 processor as compute nodes
+* 512 GB of RAM per node
+* 4 nodes with 2x Nvidia Tesla P100 GPGPUs
+* Some nodes are virtualised (several virtual machines share resources on each node)
+
+##### 1 CS400 huge memory node
+* Same Intel Xeon E5-2600 v4 processor as compute nodes
+* 4 TB of RAM per node
+
+#### Māui Compute Hardware:
+##### 464 XC50 compute nodes
+* 2x Intel Xeon Gold 6148 2.4 GHz processor with 20 physical cores each (40 physical cores per node)
+* 50% of nodes with 96 GB, 50% with 192 GB of RAM per node
+
+##### 28 CS500 nodes
+* Same Intel Xeon Gold 6148 SKL processor as compute nodes
+* 768 GB of RAM per node
+* 8 nodes with 1x Nvidia Tesla P100 GPGPU
+* Some nodes are virtualised (several virtual machines share resources on each node)
+
+#### More hardware details:
+##### CS400/CS500 nodes
+* 2-way hyperthreading with a total of 72 (CS400)/80 (CS500) logical cores per node
+* Native access to Spectrum Scale filesystems (ideal for IO-intensive jobs)
+* Non-interactive nodes for running batch jobs, e.g., preprocessing or postprocessing of HPC jobs
+* Virtual lab nodes for interactive work with remote desktop capability
+* Run a full version of CentOS 7
+
+##### XC50 compute nodes
+* 2-way hyperthreading with a total of 80 logical cores per node
+* Cray Aries interconnect (low-latency network, used by MPI jobs, IO, ...)
+* DVS (Data Virtualization Service) links for accessing Spectrum Scale filesystems over Cray Aries (currently no native access possible), which may limit IO performance for some jobs
+* Used only for running jobs, no direct access
+* Run a slim version of SLES (SUSE Linux Enterprise Server)
+
+##### XC50 eLogin nodes
+* Same Intel Xeon Gold 6148 SKL processor as compute nodes (no cross-compilation needed!)
+* Native access to Spectrum Scale filesystems
+* Used for building codes, submitting jobs, ...
+* Run on a full version of SLES
+
+#### Storage
+* Dedicated cluster for the Shared Service nodes
+* InfiniBand network for access from build nodes and CSx00 nodes to ensure high throughput and low latency
+* Cray DVS provides access to Spectrum Scale from compute nodes via the Aries interconnect
+
+#### Main improvements
+* Faster processors compared to those on FitzRoy or Pan
+* GPGPU (General Purpose Graphical Processing Unit) nodes to support science codes and visualisation
```

Comment:
Is it worth mentioning that the GPUs are much more powerful than the ones on Pan (that's a big improvement)?